sdk/core/ci.yml

```diff
@@ -21,14 +35,10 @@ pr:
     include:
     - sdk/core/
 
-resources:
-  repositories:
-    - repository: azure-sdk-tools
-      type: github
-      name: Azure/azure-sdk-tools
-      endpoint: azure
-
-jobs:
-- template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+stages:
+- template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
-    ServiceDirectory: core
\ No newline at end of file
+    ServiceDirectory: core
+    Artifacts:
+    - name: azure_core
```

Comment:
Are we sure this is the only package we want to release with this pipeline?

Reply:
Yeah - I'm not sure ... adding @lmazuel and @xiangyan99 for confirmation.

Reply:
we want to release azure-core-tracing-opencensus as well

Reply:
I have added azure-core-tracing-opencensus to the package list for the core pipeline. We can release them together or independently - it doesn't matter. Ideally everything produced by a pipeline would go out together but we've built this flexibility in.